src/bio/bio_context.c

```diff
@@ -310,7 +311,7 @@ bio_blob_delete(uuid_t uuid, struct bio_xs_context *xs_ctxt)
 	if (rc != 0)
 		return rc;
 
-	bbs = xs_ctxt->bxc_blobstore;
+	bbs = xs_ctxt->bxc_xs_blobstores[st]->bxb_blobstore;
```

Comment:
Instead of accessing the array directly, I think we'd use a helper function to get the per-xstream per-type blobstore, otherwise, so that we can support WAL, meta & data blob sharing same blobstore.

Reply:
sure